core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.directive.html

```diff
@@ -1,55 +1,56 @@
-      <div class="oppia-opportunities-subheading"><[subheading]></div>
-      <div class="oppia-opportunities-heading"><[heading]></div>
+      <div class="oppia-opportunities-subheading">{{ subheading }}</div>
+      <div class="oppia-opportunities-heading">{{ heading }}</div>
     </div>
-    <button type="button" class="close oppia-close-button-position" ng-click="confirm()" aria-label="Close">
+    <button type="button" class="close oppia-close-button-position" (click)="activeModal.close()" aria-label="Close">
       <span aria-hidden="true">&times;</span>
     </button>
   </div>
 </div>
 <div class="modal-body">
-  <div class="oppia-loading-text" ng-if="loadingData || uploadingTranslation">
+  <div class="oppia-loading-text" *ngIf="loadingData || uploadingTranslation">
     <div class="oppia-loading-text1 oppia-animate-fadein">
       <lazy-loading></lazy-loading>
     </div>
     <div class="oppia-loading-text2 oppia-animate-fadein">
       <lazy-loading></lazy-loading>
     </div>
   </div>
-  <div ng-if="!textToTranslate && !loadingData">
+  <div *ngIf="!textToTranslate && !loadingData">
     <strong>Sorry! All available translations are in review currently.</strong>
   </div>
-  <div ng-if="textToTranslate">
+  <div *ngIf="textToTranslate">
     <div class="oppia-translatable-text-section">
-      <strong>Please translate the given content in <[languageDescription]>:</strong>
-      <a class="oppia-skip-button" href="" ng-if="moreAvailable" ng-click="skipActiveTranslation()">skip</a>
+      <strong>Please translate the given content in {{ languageDescription }}:</strong>
+      <a class="oppia-skip-button" *ngIf="moreAvailable" (click)="skipActiveTranslation()">skip</a>
     </div>
-    <div ng-if="!loadingData && !uploadingTranslation" class="oppia-text-to-translate-container">
-      <angular-html-bind ng-class="{'oppia-rte-editor-focused': isCopyModeActive()}"
-                         ng-click="onContentClick($event)"
-                         html-data="textToTranslate"
-                         class="oppia-rte-editor">
-      </angular-html-bind>
+    <div *ngIf="!loadingData && !uploadingTranslation" class="oppia-text-to-translate-container">
+      <!-- textToTranslate contains the html content of an exploration state -->
+      <angular-html-bind-wrapper [ngClass]="{'oppia-rte-editor-focused': isCopyModeActive()}"
+                                 (click)="onContentClick($event)"
+                                 [htmlData]="textToTranslate"
+                                 class="oppia-rte-editor">
+      </angular-html-bind-wrapper>
     </div>
     <div class="oppia-translation-editor-section">
       <div class="protractor-test-state-translation-editor">
         <ck-editor-copy-toolbar></ck-editor-copy-toolbar>
-        <schema-based-editor schema="HTML_SCHEMA" local-value="activeWrittenTranslation.html">
+        <schema-based-editor [schema]="getHtmlSchema.bind(this)" [localValue]="activeWrittenTranslation.html">
```

Comment:
I don't fully understand why bind is required here. Is it because you want to run this Depression in schema-based-editor but in the scope of translation modal?